src/themes/masschallenge/collections/grid.overrides

```diff
@@ -65,4 +65,9 @@
 
 .ui.zero-margin.grid {
   margin: 0;
+}
+
```

Comment:
I don't see where these stylings are being used anymore. If you aren't using them anymore close the PR.